Allow Sync Frequency to be nil (#353)
fivetran/terraform-provider-fivetran: CHANGELOG.md

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [Unreleased](https://github.com/fivetran/terraform-provider-fivetran/compare/v1.2.9...HEAD)

> ## [1.2.0](https://github.com/fivetran/terraform-provider-fivetran/compare/v1.2.8...v1.2.9)

Reviewer: ```suggestion
## [1.2.9](https://github.com/fivetran/terraform-provider-fivetran/compare/v1.2.8...v1.2.9)
```

## Fixed

> - Issue with SyncFrequency unable to be set to nil

Reviewer 2: ```suggestion
- Issue with resource `fivetran_connector_schedule` when SyncFrequency is unknown.
```

We should introduce issues in change-log notes from user-understanding side. It's not clear what does this mean here, only we as a developers know that the issue caused by this fact. But from customer side it should be described in different manner.
